db/schema.rb

```diff
@@ -17,11 +17,15 @@
   enable_extension "plpgsql"
 
   create_table "games", force: true do |t|
-    t.string   "name"
+    t.integer  "game_id"
+    t.integer  "white_player"
+    t.integer  "black_player"
```

Comment:
Also, to add a column. You should be creating a migration and using `add_column`. So there should be a migration file as well.

Schema.rb is autogenerated